velox/common/memory/ByteStream.h

```diff
@@ -23,14 +23,16 @@
 namespace facebook::velox {
 
 struct ByteRange {
-  // Start of buffer. Not owned.
+  /// Start of buffer. Not owned.
   uint8_t* buffer;
 
-  // Number of bytes or bits starting at 'buffer'.
+  /// Number of bytes or bits starting at 'buffer'.
```

Comment:
Why do you say "or bits"? Is this correct?

Reply:
It is correct. StreamArena always return in bytes but ByteStream will change to number of bits. That's not part of this change. But we could consider to make the handling better next.